docs/src/pages/component-demos/drawers/UndockedDrawer.js

```diff
@@ -26,77 +26,128 @@ const styleSheet = createStyleSheet('UndockedDrawer', () => ({
     width: 250,
     flex: 'initial',
   },
-  remainder: {
-    flex: 1,
+  listFull: {
```

Comment:
Damn.  Forgot to make use it to make the ListItems full-width on the top/bottom drawers.  Update pushed.